policies/policy-edit-process.md

```diff
@@ -0,0 +1,35 @@
+General Policy Edit Process
+=============================
+
+This policy represents the way that corrective changes to policies, that don't
+change the meaning of those policies, are proposed, edited, and approved.
```

Comment:
I think it would be useful to have examples of typical changes that might fall within the scope of these simple edits, e.g. typographical, grammatical or formatting changes